test/e2e/config/e2e_conf.yaml

```diff
@@ -83,7 +83,7 @@ variables:
 
 intervals:
   default/wait-controllers: ["3m", "10s"]
-  default/wait-cluster: ["20m", "10s"]
+  default/wait-cluster: ["20m", "15m"] # The second time to check the availibility of the cluster should happen late, so kcp object has time to be created
```

Comment:
15 minutes??? that seems a long wait no?

Reply:
Yes, I want to make sure that KCP has enough time to be created. The 15m interval between two checks is still inside the range of 20 minutes maximum interval for the cluster wait check, so I think it is acceptable. 
Note that if there is no delay (it is in most of the case), the cluster object can be created and pass the first-time check, then it doesn't need to wait for 15 minutes. The 15-minute-wait only happens when there is a delay.

Reply:
10s -> 15 min seems like a bug jump to me who has little idea about the use of this code, can you explain a bit more? Would it be that if the initial check of KCP readiness fails, the test will halt until KCP is ready again? And even if the KCP was ready right after the first failed check, the test will halt 15 minutes since thats when the next check is?